docs/current_docs/manuals/developer/dependencies.mdx

```diff
@@ -0,0 +1,242 @@
+---
+slug: /manuals/developer/dependencies
+displayed_sidebar: "current"
+toc_max_heading_level: 2
+title: "Dependencies"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Dependencies
+
+When creating a Dagger module, there are two types of dependencies you will encounter:
+
+- Dependencies on other Dagger modules
+- Dependencies on third-party packages or modules from your programming language's ecosystem
+
+## Dependencies on Dagger modules
+
+Dagger modules are shareable and reusable. You can call Dagger Functions from any Dagger module in your own Dagger module simply by adding it as a module dependency with `dagger install`, as in the following example:
+
+```sh
+dagger install github.com/shykes/daggerverse/hello@v0.3.0
+```
+
+This module will be added to your `dagger.json`:
+
+```json
+...
+"dependencies": [
+  {
+    "name": "hello",
+    "source": "github.com/shykes/daggerverse/hello@54d86c6002d954167796e41886a47c47d95a626d"
+  }
+]
+```
+
+When you add a dependency to your module with `dagger install`, the dependent module will be added to the code-generation routines and can be accessed from your own module's code.
+
+The entrypoint to accessing dependent modules from your own module's code is `dag`, the Dagger client, which is pre-initialized. It contains all the core types (like `Container`, `Directory`, etc.), as well as bindings to any dependencies your module has declared.
+
+Here is an example of accessing the installed `helloWorld` module from your own module's code:
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go
+func (m *MyModule) Greeting(ctx context.Context) (string, error) {
+  return dag.HelloWorld().Message(ctx)
+}
```

Comment:
This doesn't look right :eyes: The module name installed above is `hello`, so it would be accessible at `dag.Hello()`. Same for the other languages.

The correct usage is `dag.Hello().Hello(ctx)` (trying this locally). Confused, this has clearly been wrong for a while.

Reply:
Good catch, thanks! Fixed now.